app/views/topics/show.html.erb

```diff
@@ -7,19 +7,7 @@
 <div class="row">
   <div class="col-md-8">
     <p class="lead"><%= @topic.description %></p>
-    <% @topic.posts.each do |post| %>
```

Comment:
So you would keep this loop, calling the `_post` partial for each individual `post`